force-app/main/default/classes/AuthorisationRequest.cls

```diff
@@ -79,6 +79,13 @@ public without sharing class AuthorisationRequest {
     @namespaceAccessible
     public String shopperReference { get; set; }
 
+    /**
+    * Defines a recurring payment type.
+    * @return recurringProcessingModel
+    */
+    @namespaceAccessible
+    public String recurringProcessingModel { get; set; }
```

Comment:
Should we make this an Enum with hardcoded values(`CardOnFile`, `Subscription`, `UnscheduledCardOnFile`)  instead of a String? 
https://docs.adyen.com/api-explorer/Payment/68/post/authorise3ds2#request-recurringProcessingModel

Reply:
Done